Cross compilation to ARMv7 fails

The command `env GOARCH=arm GOARM=7 go build` gives the output
```
# github.com/marcusolsson/tui-go
../../../../marcusolsson/tui-go/box.go:332:7: constant 9223372036854775807 overflows int
```
I'm using go1.9.2 linux/amd64

--- a/box.go
+++ b/box.go
@@ -2,7 +2,6 @@ package tui
 
 import (
 	"image"
-	"math"
 )
 
 var _ Widget = &Box{}
@@ -329,7 +328,7 @@ func doLayout(ws []Widget, space int, a Alignment) []int {
 
 	// Distribute remaining space
 	for {
-		min := math.MaxInt64
+		min := maxInt
 		for i, s := range sizes {
 			p := alignedSizePolicy(a, ws[i])
 			if (p == Preferred || p == Minimum) && s <= min {

--- a/grid.go
+++ b/grid.go
@@ -2,7 +2,6 @@ package tui
 
 import (
 	"image"
-	"math"
 )
 
 var _ Widget = &Grid{}
@@ -265,7 +264,7 @@ func (g *Grid) doLayout(space int, a Alignment) []int {
 	for {
 		var changed bool
 		if nonZeroStretchFactors == 0 {
-			min := math.MaxInt8
+			min := maxInt
 			for _, sz := range sizes {
 				if sz < min {
 					min = sz

--- a/math.go
+++ b/math.go
@@ -1,0 +1,5 @@
+package tui
+
+// Maximum integer value for the current architecture.
+const maxUint = ^uint(0)
+const maxInt = int(maxUint >> 1)

